packages/api-derive/src/collective/proposals.ts

```diff
@@ -3,28 +3,30 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import { ApiInterfaceRx } from '@polkadot/api/types';
+import { Option } from '@polkadot/types';
 import { Hash, Proposal, Votes } from '@polkadot/types/interfaces';
 import { DerivedCollectiveProposals } from '../types';
 
 import { combineLatest, Observable, of } from 'rxjs';
 import { map, switchMap } from 'rxjs/operators';
-import { Option } from '@polkadot/types';
 
 export function proposals (api: ApiInterfaceRx, section: 'council' | 'technicalCommittee'): () => Observable<DerivedCollectiveProposals> {
   return (): Observable<DerivedCollectiveProposals> =>
     api.query[section]
       ? api.query[section].proposals()
         .pipe(
-          switchMap((hashes: Hash[]): Observable<[Hash[], Option<Proposal>[], Option<Votes>[]]> => {
+          switchMap((hashes: Hash[] = []): Observable<[Hash[], Option<Proposal>[], Option<Votes>[]]> => {
```

Comment:
Under what conditions are hashes empty? This means there is an issue elsewhere since the query _always_ should return a `Codec` (in this case a `Vec`, which may have no elements)